golang/x/relay/types/errors.go

```diff
@@ -104,6 +104,23 @@ const (
 	NotHeavier sdk.CodeType = 405
 	// NotHeavierMessage is the corresponding message
 	NotHeavierMessage = "New best hash does not have more work than previous"
+
+	// 500-block Queries
+
+	// NotEnoughArguments means there are not enough arguments specified in the path of a query
+	NotEnoughArguments sdk.CodeType = 501
+	// NotEnoughArgumentsMessage is the corresponding message
+	NotEnoughArgumentsMessage = "Not enough arguments"
+
+	// TooManyArguments means there are too many arguments specified in the path of a query
+	TooManyArguments sdk.CodeType = 501
```

Comment:
reused code number

Reply:
Fixed.  Also realized I was returning the wrong thing in the functions below